docs/Versioning.md

```diff
@@ -26,6 +26,7 @@ For more details on ONNX Release versions, see [this page](https://github.com/on
 
 | ONNX Runtime release version | ONNX release version | ONNX opset version | ONNX ML opset version | Supported ONNX IR version | [Windows ML Availability](https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes/)|
 |------------------------------|--------------------|--------------------|----------------------|------------------|------------------|
+| 1.13.0 | **1.10** down to 1.2 | 15 | 2 | 7 | Windows AI 1.11+ |
 | 1.12.0 | **1.10** down to 1.2 | 15 | 2 | 7 | Windows AI 1.11+ |
```

Comment:
Shouldn't be the opset version number be aligned with https://github.com/onnx/onnx/blob/main/docs/Versioning.md#released-versions

Reply:
TODO - remove the table and point to https://github.com/microsoft/onnxruntime/blob/master/docs/Versioning.md#version-matrix